_inc/lib/tracks/class.tracks-client.php

```diff
@@ -157,7 +157,11 @@ static function get_anon_id() {
 
 				$anon_id = 'jetpack:' . base64_encode( $binary );
 
-				if ( ! headers_sent() ) {
+				if ( ! headers_sent()
```

Comment:
Can you please reformat the condition to look like this?
```
if (
    condition
    && condition
    && condition
) {
```
Thank you!

Reply:
Done - thanks